store/datastore/utils.go

```diff
@@ -55,11 +59,15 @@ func toList(listof []*model.RepoLite) (string, []interface{}) {
 
 // helper function that converts a simple repository list
 // to a sql IN statement compatible with postgres.
-func toListPosgres(listof []*model.RepoLite) (string, []interface{}) {
+func toListPostgres(listof []*model.RepoLite) (string, []interface{}) {
 	var size = len(listof)
-	if size > 999 {
+	switch {
+	case meddler.Default == meddler.SQLite && size > 999:
```

Comment:
You're right it does get covered since it's only in the PostgreSQL portion. I updated to not check again.